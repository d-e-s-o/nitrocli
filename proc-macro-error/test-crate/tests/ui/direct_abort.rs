extern crate test_crate;
use test_crate::make_fn;

make_fn!(direct_abort);

fn main() {}
