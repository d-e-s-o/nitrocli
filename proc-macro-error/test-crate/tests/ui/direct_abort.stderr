error: direct MacroError::abort() test
 --> $DIR/direct_abort.rs:4:10
  |
4 | make_fn!(direct_abort);
  |          ^^^^^^^^^^^^
